test/test_cli_options.py

```diff
@@ -453,6 +453,26 @@ def test_compilation(self):
         self.assertTrue(helper.has(self.compilation, 'Dry run'))
 
 
+# --classical_format string
+class TestClassicalFormat(unittest.TestCase):
+
+    def assertDryRun(self, folder, track, test):
+        self.assertEqual(helper.dry_run([
+            '--classical', '--format-classical', '$ar_combined_composer/'
+            '${ar_combined_disctrack}_%shorten{$ar_classical_title,64}_'
+            '%shorten{$acoustid_id,8}',
+            helper.get_testfile('classical', folder, track)
+        ]), test)
+
+    d = '/Debussy_Claude/'
```

Comment:
The class attribute `d = '/Debussy_Claude/'` makes only sense if there are several tests. I would write `/Debussy_Claude/01_Pagodes_.mp3`

Reply:
Well I thought for the later tests that might be making sense to keep it but I will take it out.